### PR TITLE
Move package.json#scripts from workspaces to project

### PR DESCRIPTION
**Pull request by jamiekyle-eb**

### CODE REVIEW

File: scripts/run-if.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```
Comment (jonathancreamer-eb) [inline scripts/run-if.sh]: Should this be `#!/bin/bash`? Supposedly that's what the bad substitution error we're getting in travis is from...

https://stackoverflow.com/a/23277604

**Comment by jonathancreamer-eb** [comment]
I think you can still run them package by package, but hoisting them to the top makes them run faster, and allows for a shared configuration across them. Correct me if I'm wrong @jamiebuilds?

**Comment by kwelch** [comment]
I agree that top level will be faster when ran as a whole, and you can scope the top level script to a specific package. However this removed the scripts within each package. Is that not something we expect to use when working in bolt? I have gotten use to `yarn workspace <packages> test` I assumed that would be faster if I know I only every want/need to run that package.

**Comment by jcreamer898** [comment]
@jamiebuilds ? :)

**Comment by jamiekyle-eb** [comment]
I've been wanting to find a better way to do that, I recommend just adding a `package.json` script at the root for the specific filter you want. It's easier to maintain this way

**Pull request merged**